File: Interfaz/ProyectoFinal/src/app/Parametros/Salida/AltaMejoradorOut.ts
import { ParametroBaseOut } from "./ParametroBaseOut";

export class AltaMejoradorOut extends ParametroBaseOut{

    private existeMail!: boolean;
    private existeCelular!: boolean;

    constructor()
    {
        super();
    }

    public get ExisteMail(): boolean {
        return this.existeMail;
    }
    public set ExisteMail(value: boolean) {
        this.existeMail = value;
    }
    public get ExisteCelular(): boolean {
        return this.existeCelular;
    }
    public set ExisteCelular(value: boolean) {
        this.existeCelular = value;
    }
}

File: ProyectoFinal/ProyectoFinal/Parametros/Salida/AltaMejoradorOut.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal.Parametros.Salida
{
    public class AltaMejoradorOut : ParametroBaseOut
    {
        public bool ExisteMail { get; set; }
        public bool ExisteCelular { get; set; }
    }
}